src/routes/disclaimer/containers/__tests__/DisclaimerPage.spec.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { mount, shallow } from 'enzyme'
+import { Provider } from 'react-redux'
+import configureMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+
+import ConnectedDisclaimerPage, { DisclaimerPage } from '../DisclaimerPage'
+import LanguageModel from 'modules/endpoint/models/LanguageModel'
+import DisclaimerModel from 'modules/endpoint/models/DisclaimerModel'
+import Payload from 'modules/endpoint/Payload'
+import Store from '../../../../Store'
+
+const mockSetLanguageChangeUrls = jest.fn()
+
+const location = 'augsburg'
+const languages = [
+  new LanguageModel('en', 'English'),
+  new LanguageModel('de', 'Deutsch'),
+  new LanguageModel('ar', 'Arabic')
+]
+
+const disclaimer = new DisclaimerModel({
+  id: 1689, title: 'Feedback, Kontakt und mögliches Engagement', content: 'this is a test content'
+})
+
+describe('DisclaimerPage', () => {
+  beforeEach(() => {
+    mockSetLanguageChangeUrls.mockClear()
```

Comment:
I think it would be clearer to just create a new mock function for every test. So you don't have to look up where it's created and cleared.

Reply:
hmm okay, sounds okay for me